src/components/Carousel.js

```diff
@@ -512,8 +520,9 @@ export default class Carousel extends Component {
     if (this.getProp('arrows')) {
       return (
         <button
-          className="BrainhubCarousel__arrows BrainhubCarousel__arrow-left"
+          className="BrainhubCarousel__arrows BrainhubCarousel__arrowLeft"
           onClick={this.prevSlide}
+          ref={el => this.arrowLeftNode = el}
```

Comment:
doesn't it have any side effects when the value is returned?

i.e. `el => this.arrowLeftNode = el` instead of `el => { this.arrowLeftNode = el; }`

Reply:
Doesn't matter. Return value is not used.